Sources/Vector4+nosimd.swift

```diff
@@ -125,16 +125,16 @@ extension Vector4f: Equatable {
     }
     
     public static func *(lhs: Vector4f, rhs: Matrix4x4f) -> Vector4f {
-        return Vector4f(
-            lhs.x * rhs.m11 + lhs.y * rhs.m21 + lhs.z * rhs.m31 + lhs.w * rhs.m41,
-            lhs.x * rhs.m12 + lhs.y * rhs.m22 + lhs.z * rhs.m32 + lhs.w * rhs.m42,
-            lhs.x * rhs.m13 + lhs.y * rhs.m23 + lhs.z * rhs.m33 + lhs.w * rhs.m43,
-            lhs.x * rhs.m14 + lhs.y * rhs.m24 + lhs.z * rhs.m34 + lhs.w * rhs.m44
-        )
+        return rhs.transposed * lhs
```

Comment:
since this library is more or less performance oriented I'm afraid using transpoing in multiplication is not the best approach. I'd vote for specialized implementation here

Reply:
OK, not a problem. Even more optimal would be take C library with SSE and NEON and use binding to it.